pocketlint/tests/test_restructuredtext.py

```diff
@@ -343,6 +347,39 @@ def test_check_section_delimiter_bad_marker_length(self):
         self.assertEqual(expect, self.reporter.messages)
         self.assertEqual(1, self.reporter.call_count)
 
+    def test_semantic_newline_fullstop(self):
+        """Check for a sentence with a . and no new line."""
+        content = (
+            'Sentence. New sentence\n'
```

Comment:
for this input, it would help to have some  valid semantic newlines just to make sure that we don't get false positives